egs/mgb5/s5/local/arpa2G.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
```

Comment:
could you replace the calls to this file by call to utils/format_lm.sh?

Reply:
done.

Reply:
also please remove (sorry I didn't say it before).

Reply:
done